internal/formats/common/testutils/utils.go

```diff
@@ -87,6 +89,8 @@ func AssertPresenterAgainstGoldenSnapshot(t *testing.T, pres presenter.Presenter
 	if !bytes.Equal(expected, actual) {
 		dmp := diffmatchpatch.New()
 		diffs := dmp.DiffMain(string(expected), string(actual), true)
+		t.Logf("len: %d\nexpected: %s", len(expected), expected)
```

Comment:
I see we're still using `diffmatchpath` here. 

I really love https://pkg.go.dev/github.com/google/go-cmp/cmp for these kinds of large diffs. I think the output is superior to us having to log our own `len` and do the `bytes.Equal` comparison.

The diffTesting example is pretty expressive:
https://pkg.go.dev/github.com/google/go-cmp/cmp#example-Diff-Testing

Don't have to change if you like `dmp`. I'm just throwing this library into the ring for consideration.

Reply:
I like both cmd and dmp. The upside with dmp is that you get the entire output formatted with a colorized diff. The upside with cmp is that you get a summary of the changes in a typical patch format. 

The pro to dmp is that you get the entire document which gives you full context as to where the problem might be. That is also the downside... you get the full document, which could be rather large! The pro to cmp is that it gives you a summary, but that is also it's downside... sometimes the summary doesn't provide enough context to figure the exact problem.

(I've been pretty torn on this one)